src/sequential/main.cpp

```diff
@@ -1,7 +1,10 @@
+#include <iostream>
+#include <optional>
+
 #include "sequential.hpp"
 
 
-int main(int argc, char *argv[]) {
+int main(void) {
```

Comment:
```suggestion
int main() {
```

Reply:
The `void` is actually intentional to prevent users from inputting any command line arguments.

Reply:
I don't think putting `void` there will prevent passing cmdline args. `void` and having nothing between parentheses is actually equivalent in C++ anyway since function declarations with unknown parameters are not allowed.

Reply:
Right, this isn't C.

Random stackoverflow peeps recommending using `func(void)` in C (which is not C++, so this is irrelevant. I'm just including it in case Soliman or Griffin are interested): https://stackoverflow.com/questions/693788/is-it-better-to-use-c-void-arguments-void-foovoid-or-not-void-foo